src/aks-preview/HISTORY.rst

```diff
@@ -18,6 +18,7 @@ Pending
 * Add `--enable-ai-toolchain-operator` to `az aks create` and `az aks update`.
 * Add `--disable-ai-toolchain-operator` to the `az aks update` command.
 * Refactor azure service mesh related code to meet cli style requirements.
+* Implicitly enable istio when ingress or egress gateway is enabled for Azure Service Mesh.
```

Comment:
`1.0.0b5` is already released, please leave the note under `Pending` or choose a version number (and update `VERSION` in `setup.py`)